bin/rules.rb

```diff
@@ -0,0 +1,18 @@
+# Game Rule Module for the Tic Tac Toe Game
```

Comment:
This is a good step you took here. You just need to follow through with all other stuff needed to be improved in the Game class